File: java-advent-of-code/src/test/java/advent/twenty_eighteen/Day08MemoryManeuverTest.java
package advent.twenty_eighteen;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day08MemoryManeuverTest {
    @Test
    public void checkReal() {
        Day08MemoryManeuver im = new Day08MemoryManeuver("/twenty_eighteen/Day08-MemoryManeuver-input.txt");

        assertEquals(43825, im.getPart1Answer().intValue());
        assertEquals(19276, im.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        Day08MemoryManeuver im = new Day08MemoryManeuver("/twenty_eighteen/Day08-MemoryManeuver-example#1.txt");

        assertEquals(138, im.getPart1Answer().intValue());
        assertEquals(66, im.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_eighteen/Day08MemoryManeuverTest.scala
package advent.twenty_eighteen

import org.scalatest.{FlatSpec, Matchers}

class Day08MemoryManeuverTest  extends FlatSpec with Matchers {
  "2018 Day 08 - Input" should "calculate answers" in {
    val dp = new Day08MemoryManeuver("/twenty_eighteen/Day08-MemoryManeuver-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day08MemoryManeuver("/twenty_eighteen/Day08-MemoryManeuver-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
